node-playwright-chrome/Dockerfile

```diff
@@ -28,15 +28,14 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update \
     fonts-kacst \
     fonts-thai-tlwg \
     fonts-wqy-zenhei \
-    --no-install-recommends \
-    && rm -rf /var/lib/apt/lists/* \
-    && rm -rf /src/*.deb \
-    && mkdir -p /tmp/.X11-unix \
-    && chmod 1777 /tmp/.X11-unix
```

Comment:
maybe we can do at least some of those here? like removing the `lists`?

Reply:
But it will affect the image size, as it will become part of this layer, right?

Reply:
I can check how much space the image takes up with this versus inlining these calls in different layers tomorrow